bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -49,6 +49,9 @@ spec:
           #!/bin/bash
           set -euo pipefail
 
+          # enable ip forwarding
+          sysctl -w net.ipv4.ip_forward=1
```

Comment:
Since RHCOS only exists to support OpenShift, should it enable this itself by default?

Reply:
Probably, yes. This was in openshift-ansible before, so I moved it to the operator.

I'm looking for a reasonable rhcos package to put this in.